uberfire-extensions/uberfire-runtime-plugins/uberfire-runtime-plugins-api/src/main/java/org/uberfire/ext/plugin/type/SplashPluginResourceTypeDefinition.java

```diff
@@ -35,4 +46,9 @@ public String getDescription() {
     public String getSuffix() {
         return "/" + PluginType.SPLASH.toString().toLowerCase() + ".plugin";
     }
+
+    @Override
+    public Category getCategory() {
+        return null;
```

Comment:
What's the purpose of the field `private Category category` and initializing it in the constructor, when you're always returning null here?

Reply:
@jhrcek my mistake!